components/items/ItemsGrid/ItemsGrid.vue

```diff
@@ -111,6 +111,11 @@ const props = defineProps<{
   collectionPopoverHide?: boolean
 }>()
 
+// display sn if nftId is less than 4 digits
+const displaySn = (nftId) => {
+  return props.displayNameWithSn && nftId.split('-')[1].length <= 4
+}
+
```

Comment:
It's not enough to applying it on `ItemsGrid`.  Let's move this check into utils function, to avoid showing different SN on different pages.

ref: `utils/nft.ts -> nameWithIndex`

Reply:
> ref: `utils/nft.ts -> nameWithIndex`

thank you, updated